articles/quickstart/native/android/00-centralized-login.md

```diff
@@ -0,0 +1,115 @@
+---
```

Comment:
Can we add something to "wrap up" this step of the quickstart? For example, show an image of the hosted Lock in a phone frame. There's a phone frame in the styleguide that can be used, just need to apply the class to the image: 
```
<div class="phone-mockup">
  <img src="..." alt="Mobile example screenshot" />
</div>
```